projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java

```diff
@@ -358,56 +314,62 @@ public static void saveTrips(Path csvPath, String path) {
-            .setDirectionId(Integer.parseInt(record.get("direction_id")))
-            .setShapeId(record.get("shape_id"))
-            .build()
-            .writeDelimitedTo(output);
+        trips.add(
+            trip.setRouteId(Integer.parseInt(record.get("route_id")))
+                .setServiceId(Integer.parseInt(record.get("service_id")))
+                .setTripId(record.get("trip_id"))
+                .setTripHeadsign(record.get("trip_headsign"))
+                .setDirectionId(Integer.parseInt(record.get("direction_id")))
+                .setShapeId(record.get("shape_id"))
+                .build());
       }
+      return trips;
     } catch (IOException e) {
       throw new RuntimeException(e);
     }
   }
 
+  public static List<String> getStopsName() {
+    return getStops(absStops).stream().map(Stop::getStopName).collect(Collectors.toList());
+  }
+
+  // Paths to the CSVs files
+  static Path absCalendar;
+  static Path absRoutes;
+  static Path absStopTimes;
+  static Path absStops;
+  static Path absTranslations;
+  static Path absTripsTxt;
+
   public static void main(String[] args) {
-    // Paths to the CSVs files
-    final Path agencyTxt = Paths.get(args[0]);
-    final Path calendarTxt = Paths.get(args[1]);
-    final Path fareAttributesTxt = Paths.get(args[2]);
-    final Path fareRulesTxt = Paths.get(args[3]);
-    final Path routesTxt = Paths.get(args[4]);
-    final Path shapesTxt = Paths.get(args[5]);
-    final Path stopTimesTxt = Paths.get(args[6]);
-    final Path stopsTxt = Paths.get(args[7]);
-    final Path translationsTxt = Paths.get(args[8]);
-    final Path tripsTxt = Paths.get(args[9]);
 
-    // Paths to the output files
-    final String agencyOutputPath = args[args.length - 10];
-    final String calendarOutputPath = args[args.length - 9];
-    final String fareAttributesOutputPath = args[args.length - 8];
-    final String fareRulesOutputPath = args[args.length - 7];
-    final String routesOutputPath = args[args.length - 6];
-    final String shapesOutputPath = args[args.length - 5];
-    final String stopTimesOutputPath = args[args.length - 4];
-    final String stopsOutputPath = args[args.length - 3];
-    final String translationsOutputPath = args[args.length - 2];
-    final String tripsOutputPath = args[args.length - 1];
+    {
+      absCalendar = Paths.get(args[0]);
+      absRoutes = Paths.get(args[1]);
+      absStopTimes = Paths.get(args[2]);
+      absStops = Paths.get(args[3]);
+      absTranslations = Paths.get(args[4]);
+      absTripsTxt = Paths.get(args[5]);
+    }
+
+    List<Protos.Calendar> calendarMessages = getCalendar(absCalendar);
+    List<Route> routeMessages = getRoutes(absRoutes);
+    List<StopTime> stopTimeMessages = getStopTimes(absStopTimes);
+    List<Stop> stopMessages = getStops(absStops);
+    List<Translation> traslationMessages = getTranslations(absTranslations);
+    List<Trip> tripMessages = getTrips(absTripsTxt);
+    getStopsName();
```

Comment:
The result for `getStopsName()` is not used.

Reply:
This method is used to be one of the API methods. I add more methods to the API, for any application who want to interact with those data. Is that correct way to do?

Reply:
Sorry, I don't understand - why are you calling this method here, if you are not using its result?

Reply:
I'll remove this statement, It was for debugging. The method `getStopName()` will be called whenever the application needs.